apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.IntegrationTests/BigQueryMLFixture.cs

```diff
@@ -26,12 +24,19 @@ public class BigQueryMLFixture : CloudProjectFixtureBase, ICollectionFixture<Big
         public string DatasetId { get; }
         public string ModelId { get; }
 
+        public string StorageBucketName { get; }
+
+        public StorageClient StorageClient { get; }
+
         public BigQueryMLFixture()
         {
             DatasetId = IdGenerator.FromDateTime(prefix: "testml_");
             ModelId = CreateModelId();
+            StorageClient = StorageClient.Create();
 
             CreateData();
+            StorageBucketName = IdGenerator.FromDateTime(prefix: "bigquerytestsml-");
```

Comment:
Please add this prefix to https://github.com/googleapis/google-cloud-dotnet/blob/master/apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.CleanTestData/Program.cs

Reply:
(Or just use bigquerytests- as the prefix.)